### ISSUE TITLE
Glossary sort should be case-insensitive

### ISSUE THREAD

**Issue by shimizukawa** [sphinx-doc/sphinx#167]
Following glossary -example sorts the **ccc** in the **end of the list** while the expected ordering is: **AAA,BBB,ccc,DDD**

```
.. glossary::
   :sorted:

   BBB
      Description of BBB

   ccc
      Description of ccc

   DDD
      Description of DDD

   AAA
      Description of AAA
```

---
- Bitbucket: https://bitbucket.org/birkenfeld/sphinx/issue/167
- Originally reported by: Juha Mustonen
- Originally created at: 2009-05-13T21:35:56.088

**Comment by shimizukawa** [comment]
_From Georg Brandl on 2009-06-16 20:08:47+00:00_

Thanks, fixed in <<cset 5e927d724dbe3650356c50f5fb44c85ba7ac9aab>>.

**Comment by shimizukawa** [comment]
_From Georg Brandl on 2013-03-31 13:57:09+00:00_

Removing version: 0.6 (automated comment)
